apps/web/sanity/schemaTypes/postType.ts

```
@@ -5,7 +5,7 @@ export const postType = defineType({
   name: "post",
   title: "Post",
   type: "document",
-  icon: DocumentTextIcon,
+  icon: DocumentTextIcon as any,
```

Biome

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)